src/cascade/executor/epiviz_runner.py

```diff
@@ -116,9 +130,17 @@ def add_omega_constraint(model_context, execution_context, sex_id):
 
     children = get_descendents(execution_context, children_only=True)  # noqa: F841
     children_asdr = asdr.query("node_id in @children")
+    # Transform the children to be the random effect for the rate.
+    parent_value = parent_asdr[["age_lower", "time_lower", "mean"]].rename({"mean": "parent_mean"}, axis=1)
+    parent_and_child = children_asdr.merge(parent_value, how="left", on=["age_lower", "time_lower"])
+    # This could result in an Inf value, but that's a legal value, so...
+    children_effects = parent_and_child.assign(
```

Comment:
Oh, nice. This is less complicated than I'd feared.

Reply:
It gets simple when parent and effects are on the same grid, and we defined them, which puts them on the same grid.